http/jsonapi/runtime/standart_params.go

```diff
@@ -102,16 +154,21 @@ func FilterFromRequest(r *http.Request, mapper Mapper) (QueryOption, error) {
 				filterValues = append(filterValues, strings.Split(val, ",")...)
 			}
 			if len(filterValues) == 1 {
-				query.Where(name+"=?", filterValues[0])
+				query.Where("?=?", name, filterValues[0])
 				continue
 			}
-			query.Where(name+" IN (?)", strings.Join(filterValues, ","))
+			query.Where("? IN (?)", name, strings.Join(filterValues, ","))
 		}
 		return query
-	}, nil
+	}
+
+	if len(invalidFilter) != 0 {
+		return filterQueryOption, fmt.Errorf("at least one filter parameter is not valid: %q", strings.Join(invalidFilter, ","))
+	}
+	return filterQueryOption, nil
 }
 
 // DefaultMapper maps an query field directly name to the same name as database column name
-func DefaultMapper(in string) string {
-	return in
+func DefaultMapper(in string) (string, bool) {
```

Comment:
so should we have a mapper now, that is getting a list and returns true if something is in the list? maybe even a map ...

Reply:
a Map only allows a explicit whitelist of parameters
a function allows something like a regex to check if the fieldname is valid
=> Both an option, a map is more restrictive by default. I will change it to use a map